internal/operators/cnv/cnv_operator_test.go

```diff
@@ -207,7 +208,7 @@ var _ = Describe("CNV operator", func() {
 			host := models.Host{Role: models.HostRoleMaster}
 			haMode := models.ClusterHighAvailabilityModeNone
 			cluster = common.Cluster{
-				Cluster: models.Cluster{HighAvailabilityMode: &haMode},
+				Cluster: models.Cluster{HighAvailabilityMode: &haMode, OpenshiftVersion: lvm.LvmMinOpenshiftVersion},
```

Comment:
Should we maybe include some unit testing that ensures LVMO got installed instead of HPP?

Reply:
I'll add the uni test

Reply:
added the unit test